src/providers/Gardens.js

```diff
@@ -23,30 +23,26 @@ import { getGardenForumUrl } from '../utils/garden-utils'
 import useGardenFilters from '@/hooks/useGardenFilters'
 import { testNameFilter } from '@/utils/garden-filters-utils'
 import { useDebounce } from '@/hooks/useDebounce'
+import { useMounted } from '@/hooks/useMounted'
 
 import { getVoidedGardensByNetwork } from '../voided-gardens'
+import { getGarden } from '@1hive/connect-gardens/dist/cjs/gardens'
```

Comment:
We should update this line to export directly from `@1hive/connect-gardens`.

Reply:
Oops, I completely forgot about removing this when testing the connector locally.